src/sentry/static/sentry/app/styles/spacingScale.jsx

```diff
@@ -0,0 +1,28 @@
+export default size => {
+  if (size > 9) return (size - 9) * 100;
```

Comment:
What is this for?

Reply:
it's for big numbers. basically if you do something like `width: spacingScale(10)` it would return `100` and if you did `spacingScale(12)` it would return 300 and so on. 

I'm not attached to this, in fact most of the time i like to just keep big numbers close to home. we could just throw an error if someone puts in a number that is too big?